### ISSUE TITLE
Implement process inspector

### ISSUE THREAD

**Issue by AkihiroSuda** [osrg/earthquake#104]
Port the PoC written in Python into Go: https://github.com/AkihiroSuda/MicroEarthquake/tree/v20160203

ETA: this February

**Comment by AkihiroSuda** [comment]
#105 implemented this.

Usage: `earthquake inspectors proc -root-pid 42 -autopilo foobar.toml`
`foobar.toml`:
```toml
explorePolicy = "random"

[explorePolicyParam]
# can be empty
```

Also, `earthquake-container` utility will be implemented after #103.

**Comment by AkihiroSuda** [comment]
#107 closes this.
